src/library_dylink.js

```diff
@@ -304,8 +304,10 @@ var LibraryDylink = {
             };
           }
           // otherwise this is regular function import - call it indirectly
+          var resolved;
           return obj[prop] = function() {
-            return resolveSymbol(prop, 'function', true).apply(null, arguments);
+            if (!resolved) resolved = resolveSymbol(prop, 'function', true);
```

Comment:
can this be outside of the `function()`, 2 lines earlier? or is it resolved on the actual first call? a comment might help if so.

Reply:
No the idea is that resolution is not done until the function is first called.  If we try to resolve now most of symbol would be missing.